docs/data/data-grid/filtering/CustomFilterPanelContent.js

```diff
@@ -69,6 +69,10 @@ export default function CustomFilterPanelContent() {
                 size: 'small',
                 sx: { mt: 'auto' },
               },
+              InputComponentProps: {
```

Comment:
I wonder if we can reflect `FilterFormValueInput > currentOperator.InputComponent` hierarchy in here?
I would expect something like this:

```ts
filterFormProps: {
  valueInputProps: {
    // will be passed to `FilterFormValueInput`
    // ...
    
    InputComponentProps: {
      // will be passed to `currentOperator.InputComponent`
    }
  }
}
```

Especially since `InputComponentProps` are only passed to value input, not to other inputs

Reply:
done